package/doc/sphinx/source/documentation_pages/analysis_modules.rst

```diff
@@ -69,6 +69,14 @@ Hydrogen bonding
    analysis/hbond_analysis
    analysis/hbond_autocorrel   
 
+Water bridge
```

Comment:
Don't create a new section but add it under [*Hydrogen bonding*](https://www.mdanalysis.org/docs/documentation_pages/analysis_modules.html#hydrogen-bonding)

Reply:
@orbeckst Thanks, changed.